llvm/test/Instrumentation/MemorySanitizer/ARM32/vararg-arm32.ll

```diff
@@ -1,8 +1,8 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
-; RUN: opt < %s -S -passes=msan 2>&1 | FileCheck %s
+; RUN: opt < %s -S -passes=msan -msan-origin-base=0x40000000 -msan-and-mask=0x80000000 2>&1 | FileCheck %s
```

Comment:
Can we remove those and add proper consts into msan pass?

Reply:
We can, but this will require some research on where to put shadow and origin. Currently, I cannot say what their proper addresses will be.

Reply:
Then I afraid if someone decide to fix them will forget to update tests.
Can you put a comment in MemoryMapParams for corresponding platforms // FIXME: Remove -msan-origin-base -msan-and-mask added by #109284 after picking good constants.